docs/wiki/deployment/yara.md

```diff
@@ -1,5 +1,7 @@
 # YARA-based scanning with osquery
 
+YARA is a tool that allows you to scan files for malware (or just about anything you might need to find) using textual or binary patterns. 
```

Comment:
YARA is a tool that allows you to find textual or binary patterns inside of files.

Reply:
Change made, thank you!